src/compiler/checker.ts

```diff
@@ -13300,6 +13300,22 @@ namespace ts {
                         continue;
                     }
                 }
+                if (isInJSFile(decl) && decl.jsDoc) {
+                    let hasJSDocOverloads = false;
+                    for (const node of decl.jsDoc) {
+                        if (node.tags) {
+                            for (const tag of node.tags) {
+                                if (isJSDocOverloadTag(tag)) {
+                                    result.push(getSignatureFromDeclaration(tag.typeExpression));

```

Comment:
Operations that go from a signature back to a declaration might fail if the signature is a JSDocSignature. But (1) JSDocCallbackTag already exists, and has shaken out those bugs. (2) I can't think of any specific places; the only new ones would be ones that operate on symbols with *multiple* signatures.

Reply:
Is there anything in particular here that you would like me to look into?